examples/tests/keyboard_hid/main.c

```diff
@@ -0,0 +1,35 @@
+#include <string.h>
+
+#include <button.h>
+#include <usb_keyboard_hid.h>
+
+
+uint8_t keyboard_buffer[64];
+
+// Callback for button presses.
+//   btn_num: The index of the button associated with the callback
+//   val: 1 if pressed, 0 if depressed
```

Comment:
Shouldn't `val` be checked to make sure the button was released?

Reply:
I think this is fine as-is. Spurious GPIO activations aren't critical to avoid here.